certbot-apache/MANIFEST.in

```diff
@@ -1,8 +1,8 @@
 include LICENSE.txt
 include README.rst
-recursive-include tests *
 recursive-include certbot_apache/_internal/augeas_lens *.aug
 recursive-include certbot_apache/_internal/tls_configs *.conf
+recursive-include certbot_apache/_internal/tests/testdata *
```

Comment:
Is `certbot_apache/_internal/tests/apache-conf-files` omitted intentionally?

I also notice that `certbot_apache/_internal/augeas_lens/README` is omitted, though that's not new here.

Reply:
Yeah I did that on purpose. We didn't use to ship it, but we started as a side effect of https://github.com/certbot/certbot/pull/7607 and I don't think it's something we should do. Those tests are super hacky and `apache-conf-test-pebble.py` relies on things like `certbot-ci` which we have never shipped.